### PR TITLE
More Spanish Translation

### PR DESCRIPTION
**Pull request by Cashiot**
More translation to match the game content

### CODE REVIEW

File: application/src/main/resources/locale/application.csv [hunk 1]
```diff
@@ -1,115 +1,115 @@
...
-segment.type.hardpoint_absolute,Absolute,,,,,,Абсолютный
-segment.type.hardpoint_caustic,Caustic,,,,,,Едкий
-segment.type.hardpoint_antixeno,Anti-xeno,,,,,,АИ
-segment.type.power_group_none,Available,,,,,,Доступно
-segment.type.power_overpower,Power deficit,,,,,,Энергодефицит
-tabs.onfoot,On foot,,,,,,Пеший режим
-tabs.ships,Ships,,,,,,Корабельный режим
-notification.type.hge,High grade emission,,,,,,Высокоуровневый сигнал
-tabs.hge,HGE Tracker,,,,,,Трекер ВУС
+messageKey,,en,de,es,fr,pt,ru,,,,
```
Comment (jixxed) [inline application/src/main/resources/locale/application.csv:1]: these extra columns shouldn't be here

**Comment by Cashiot** [comment]
It should be solved. I don't know why they appeared.

File: application/src/main/resources/locale/application.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 messageKey,,en,de,es,fr,pt,ru
-blank,,,,,,,
+blank,,,,,,,,
```
Comment (jixxed) [inline application/src/main/resources/locale/application.csv:2]: still an extra comma

File: application/src/main/resources/locale/application.csv [hunk 3]
```diff
@@ -78,38 +78,38 @@ notification.type.irrelevant_pickup,Irrelevant Pickup,Irrelevant Pickup,Irreleva
...
-segment.type.power_group_none,Available,,,,,,Доступно
-segment.type.power_overpower,Power deficit,,,,,,Энергодефицит
-tabs.onfoot,On foot,,,,,,Пеший режим
-tabs.ships,Ships,,,,,,Корабельный режим
-notification.type.hge,High grade emission,,,,,,Высокоуровневый сигнал
-tabs.hge,HGE Tracker,,,,,,Трекер ВУС
+statusbar.api.stale,Fleetcarrier data might be stale! Link your Frontier account,Fleetcarrier data might be stale! Link your Frontier account,,Los datos de su carguero pueden estar obsoletos. Vincula tu cuenta de Frontier.,,,Данные о флотоносце могут быть устаревшими! Подключите аккаунт Frontier.
+segment.type.present,Present,,,Presente,,,Присутствует
+segment.type.not_present,Not present,,,No presente,,,Не присутствует
+segment.type.power_group_p,P,,,,,,,
```
Comment (jixxed) [inline application/src/main/resources/locale/application.csv:97]: still an extra comma for power groups

**Review by jixxed — CHANGES_REQUESTED** [review]

**Comment by Cashiot** [comment]
Removed that commas

**Review by jixxed — APPROVED** [review]

**Comment by jixxed** [comment]
thnx 😄

**Pull request merged**